src/sentry/issues/endpoints/group_events.py

```diff
@@ -91,12 +92,12 @@ def _get_events_snuba(
     ) -> Response:
         default_end = timezone.now()
         default_start = default_end - timedelta(days=90)
-        params = {
-            "project_id": [group.project_id],
-            "organization_id": group.project.organization_id,
-            "start": start if start else default_start,
-            "end": end if end else default_end,
-        }
+        params = ParamsType(
```

Comment:
~generally `TypedDict`s shouldn't be constructed (since they're slower at runtime) -- this should instead be `params: ParamsType = { ... }`

Reply:
TIL!